### ISSUE TITLE
Bug: empty-line-between-blocks doesn't detect top level blocks

### ISSUE THREAD

**Issue by segovia94** [sasstools/sass-lint#54]
I can't get any warning on top level selectors missing empty lines. Only selectors inside parent blocks are being detected.
```
// No warning thrown
.element {
  color: red;
}
.sibling {
  color: blue;
}
```
```
// This does correctly throw a warning
.block {
  
  .element {
    color: red;
  }
  .sibling {
    color: blue;
  }

}
```
I'm running it through the grunt plugin 0.1.0-beta.3 which is using sass-lint v1.0.0.

**Comment by segovia94** [comment]
I just found in the [docs](https://github.com/sasstools/sass-lint/blob/master/docs/rules/empty-line-between-blocks.md) that this is actually designed to only do nested blocks. Is there any reason we shouldn't also allow it to do top level blocks as well?

**Comment by Snugug** [comment]
Only cause it was written that way. I think it's fair that it should test all blocks
